### PR TITLE
added check_file_status() for input_table

### PR DESCRIPTION
**Pull request by safay**
Added file status checking `check_file_status()` for args.input_table in filter_abund.py
Not a documented issue, but found it while examining code for another issue.

- [x] Is it mergable?
- [x] Did it pass the tests?
- [x] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage.
- [x] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the ChangeLog?
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by safay** [comment]
@mr-c:
ready for your review!

File: ChangeLog [hunk 1]
```diff
@@ -3,6 +3,10 @@
    * scripts/abundance-dist.py: disambiguate documentation for force and 
    squash options
 
+2015-04-13 Scott Fay <scott.a.fay@gmail.com>
```
Comment (mr-c) [inline ChangeLog]: At the top of the file, please :-)
Two spaces between your name and the date; two spaces between your name and your bracketed email address. Thanks!

**Comment by mr-c** [comment]
Otherwise, this looks good to me!

**Comment by safay** [comment]
@mr-c: okay, changed to the true change history, sorry about that!

**Comment by mr-c** [comment]
Jenkins, ok to test

**Comment by mr-c** [comment]
@safay Congratulations on your first commit to the khmer project! Your name will be included in the release notes for the next version and you'll be listed amongst our other contributors in the next software release paper.

**Pull request merged**